src/config.ts

```diff
@@ -94,6 +94,7 @@ export const FIATCONNECT_NETWORK =
 export const STATSIG_ENV = {
   tier: DEFAULT_TESTNET === 'mainnet' ? 'production' : 'development',
 }
+export const DEFAULT_STATSIG_ID = 'default_statsig_id'
```

Comment:
kind of a nit but I think this is a bit misleading. The only case we want this to be a user's statsig ID is if it's an e2e test. Seems safer to explicitly include e2e_test in the variable name.